integration/messaging/messaging_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package messaging
+
+import (
+	"flag"
+	"log"
+	"os"
+	"regexp"
+	"testing"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"firebase.google.com/go/integration/internal"
+	"firebase.google.com/go/messaging"
+)
+
+const testRegistrationToken = "fGw0qy4TGgk:APA91bGtWGjuhp4WRhHXgbabIYp1jxEKI08ofj_v1bKhWAGJQ4e3arRCWzeTfHaLz83mBnDh0a" +
```

Comment:
Please add a comment indicating this is not used.

Reply:
Done